zebra-chain/src/transaction/serialize.rs

```diff
@@ -1325,3 +1351,174 @@ impl FromHex for SerializedTransaction {
         Ok(bytes.into())
     }
 }
+
+// TODO: After tx-v6 merge, refactor to share common serialization logic with V5.
+// For now the only difference is versioned signatures in V6.
```

Comment:
The line is incorrect, there are other changes. Let's remove completely (keep `TODO: ..`)